src/chocolatey.console/Program.cs

```diff
@@ -370,5 +374,24 @@ private static void write_warning_for_incompatible_versions()
 Or by passing the --skip-compatibility-checks option when executing a
 command.");
         }
+
+        private static void check_installed_dotnetfx_version()
+        {
+            if (Platform.get_platform() == PlatformType.Windows)
+            {
+                const string REGKEY = @"SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full\";
+                const int NET48RELEASEBUILD = 528040;
+
+                using (var ndpKey = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry32).OpenSubKey(REGKEY))
+                {
+                    if (ndpKey == null || ndpKey.GetValue("Release") == null || (int)ndpKey.GetValue("Release") < NET48RELEASEBUILD)
+                    {
+                        throw new ApplicationException(
+                            ".NET 4.8 is not installed or may need a reboot to complete installation{0}Install .NET 4.8 and reboot the system"

```

Comment:
Can this use similar wording to what is done here: https://github.com/chocolatey/choco/pull/2933/files#diff-21bdb1742369a87a1e2f23b0c59b77df037fe5d3ded2862a14cef9be2bfd13e6R788 including the link to the download location?

Reply:
I've updated the wording, hopefully it looks better?

Reply:
I've updated it to include the download URL.